plugin.json

```
@@ -59,6 +59,22 @@
             "type": "string",
             "default": "",
             "required": false
+        },
+        {
+            "key": "companiesGroupType",
+            "hint": "Group type that you want to enrich with hubspot companies",
+            "name": "Company Group Type",
+            "type": "string",
+            "default": null,
```

if type is `string` let's give this a `string` default
```suggestion
            "default": "",
```